src/js/views/hallOfFame.js

```diff
@@ -43,7 +60,32 @@ async function updatePlayers(inputs, updateEvents) {
     }
 }
 
-export default bbgmViewReact.init({
+function uiFirst(vm) {
```

Comment:
I'm deleting this block entirely. We can revisit "Remembered as..." legacy as a separate change in the future.